pmctl

```diff
@@ -124,7 +135,14 @@ build_pagemill() {
 }
 
 run_pagemill() {
-    echo "*** RUN ***"
+    echo "*** RUN ***" 
+    # Create the media path before the container is run
+    cd ${PAGEMILL_PATH}
+    if [ ! -d "static/media" ]; then
+        mkdir -p static/media;
+    fi
+    cp -r media_files/media static/
+    cd -
```

Comment:
Are these statements for creating static/media directory and copying media files over already done as part of main pmctl code line as shown above? If so, they will not be needed here.